tools/run-materialize.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+dl_materialized () {
+	INSTALL_DIR=$1
+	git -C $INSTALL_DIR clone git@github.com:MaterializeInc/materialize.git
+	cd $INSTALL_DIR"/materialize"
+}
+
+edit_rp_service () {
+	SERVICE_FILE_PY=$1"/materialize/misc/python/materialize/mzcompose/services.py"
+
+	TARGET_VERSION=$2
+	echo "Using RP version: $TARGET_VERSION"
+
+	DEFAULT_RP_VERSION="v21.11.13"
+	sed -i'' "s@$DEFAULT_RP_VERSION@$TARGET_VERSION@" $SERVICE_FILE_PY
+
+	STABLE=$3
+	if [ "$STABLE" = "unstable" ]
+	then
+		echo "Using unstable repo"
+		sed -i'' "s@vectorized/redpanda@vectorized/redpanda-unstable@" $SERVICE_FILE_PY
+	fi
```

Comment:
Can this be made idempotent, since it might be useful to be able to check a beta and then the release. Unless teardown undoes the change.

Reply:
Oh, right. Can that be optional?